### PR TITLE
Enable BGPDisco

### PR DESCRIPTION
**Pull request by spolack**

### CODE REVIEW

File: roles/cfg_openwrt/templates/corerouter/config/olsrd.j2
Comment (pktpls) [inline roles/cfg_openwrt/templates/corerouter/config/olsrd.j2:7]: Is it possible that this resets the node lifetime on hopglass?

Comment (Noki) [reply roles/cfg_openwrt/templates/corerouter/config/olsrd.j2:7]: No, with my change (https://github.com/freifunk-berlin/falter-packages/commit/ef840094613e427d9e189e4a7284248d63bc2873) we strip the suffix and force it to .olsr afterwards, however it has still implications as the hostname displayed by hopglass will not work anymore. In addition we need to update the mass-update script and there is probably e.g. documentation that becomes outdated, and hardcoded links of olsr services might stop working, ...

Comment (PolynomialDivision) [reply roles/cfg_openwrt/templates/corerouter/config/olsrd.j2:7]: For me the implications are fine. I see them not as important to block the bgpdisco improvements that I see as enormous helpful.

**Review by PolynomialDivision — APPROVED** [review]

File: roles/cfg_openwrt/templates/corerouter/config/olsrd.j2
Comment (Noki) [reply roles/cfg_openwrt/templates/corerouter/config/olsrd.j2:7]: A small addition: bird seems to work fine on mela-core (arm_cortex-a7_neon-vfpv4) without bgbdisco, but with it bird is stuck in a crash loop and mela-core is only reachable via olsr. For me this is a showstopper and should be resolved before merging this.

**Comment by pktpls** [comment]
LGTM, but can't merge as long as the Bird / armv7 crash isn't resolved

**Comment by pktpls** [comment]
Sorry, ich hatte kurz den Überblick verloren - der Bird / armv7 Crash hat nix mit BGPdisco zutun

**Comment by PolynomialDivision** [comment]
So the armv7 fix is merged, so we could also probably merge this?

**Comment by pktpls** [comment]
Coolster Hack :)

**Pull request merged**
